htdocs/takepos/invoice.php

```diff
@@ -680,6 +680,8 @@ function fail($message)
 			} else {
 				if (empty($user->rights->takepos->editlines) || (empty($user->rights->takepos->editorderedlines) && $line->special_code == "4")) {
 					dol_htmloutput_errors($langs->trans("NotEnoughPermissions", "TakePos"), null, 1);
+				} elseif ($conf->global->TAKEPOS_CHANGE_PRICE_HT == 1) {
```

Comment:
Try to use   getDolGlobalString('TAKEPOS_CHANGE_PRICE_HT')  instead of  $conf->global-TAKEPOS_CHANGE_PRICE_HT
so it will be compatible with php8.

Reply:
hello @eldy do youmean that each time we use $conf->global in the core and external modules, we have to call this function instead ???

Reply:
Yes, this is the new way to read configurations from develop/v15.